hipsparse/PKGBUILD

```diff
@@ -1,13 +1,13 @@
 # Maintainer: Markus Näther <naetherm@informatik.uni-freiburg.de>
 pkgname=hipsparse
 pkgver=3.3.0
-pkgrel=2
+pkgrel=4
```

Comment:
Shouldn't this be `pkgrel=3`?

Reply:
3 was the previous commit that fixes compile. 4 is this commit that removes hip makedepends.

Reply:
Can you leave it at 3 since when we merge this it will squashed and merged as one commit.

Reply:
Ok. Updated to 3 now.